lib/exercises.rb

```diff
@@ -1,19 +1,55 @@
 
 # This method will return an array of arrays.
 # Each subarray will have strings which are anagrams of each other
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: O(n)
+# Space Complexity: O(n)
 
-def grouped_anagrams(strings)
-  raise NotImplementedError, "Method hasn't been implemented yet!"
+def grouped_anagrams(strings) 
```

Comment:
The time complexity is O(n) if the words are of limited length, otherwise it's O(n m log m) where m is the length of the words.